packages/uops-dashboard/tailwind.config.ts

```diff
@@ -25,4 +25,5 @@ const config: Config = {
     },
   ],
 }
+// biome-ignore lint/style/noDefaultExport: <explanation>
```

Comment:
<img width="470" alt="image" src="https://github.com/user-attachments/assets/58ee7629-583c-432a-a514-888b37de7067">

Reply:
Fixed, was not referencing root biome config file